packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx

```diff
@@ -12,7 +12,13 @@ type AttachmentImagePreviewProps = {
   imageObjectFit?: 'contain' | 'cover';
 };
 
-const AttachmentImagePreview = ({ label, imagePreview, meta, imageObjectFit }: AttachmentImagePreviewProps) => {
+const defaultProps: Partial<AttachmentImagePreviewProps> = {
+  meta: undefined,
+  imageObjectFit: 'cover',
+};
+
+const AttachmentImagePreview = (props: AttachmentImagePreviewProps) => {
+  const { label, imagePreview, meta = defaultProps.meta, imageObjectFit = defaultProps.imageObjectFit } = props;
```

Comment:
why is it here like this and not like in other components?

```const propsWithDefaults = { ...defaultProps, ...props };```

Reply:
because TS hates me 🤷

Reply:
i think you need to move `AttachmentImagePreviewProps` to types file and then it should be working